app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2792,7 +2792,7 @@ private void displayCurrentInfo() {
         if (sensor_age > 0 && (DexCollectionType.hasLibre() || hasLibreblock())) {
             final String age_problem = (Pref.getBooleanDefaultFalse("nfc_age_problem") ? " \u26A0\u26A0\u26A0" : "");
             if (Pref.getBoolean("nfc_show_age", true)) {
-                sensorAge.setText(getResources().getQuantityString(R.plurals.sensor_age, sensor_age, JoH.qs(((double) sensor_age) / 1440, 1)) + age_problem);
+                sensorAge.setText(getResources().getQuantityString(R.plurals.sensor_age, sensor_age / 1440, JoH.qs(((double) sensor_age) / 1440, 1)) + age_problem);
```

Comment:
Did you test this expression?

Reply:
The int division is necessary for `getQuantityString` to select the correct plural.